.github/workflows/build-push-docker.yaml

```
@@ -17,6 +17,12 @@ on:
     tags:
       - "*"
 
+env:
+  DOCKER_ORG: aktech
```

```suggestion
  DOCKER_ORG: nebari
```
I assume this was unintentional ;)

---

Ah, thanks for noticing this. 😅